In-house patch to update the generate_sample script and related data
files for use with Solaris.

--- nova-2015.1.2/nova/openstack/common/config/generator.py.~1~	2015-10-13 07:52:44.000000000 -0700
+++ nova-2015.1.2/nova/openstack/common/config/generator.py	2016-01-31 21:59:34.000000000 -0800
@@ -104,7 +104,7 @@ def generate(argv):
     # and the list of Opt instances for that group.
     if parsed_args.libraries:
         loader = stevedore.named.NamedExtensionManager(
-            'oslo_config.opts',
+            'oslo.config.opts',
             names=list(set(parsed_args.libraries)),
             invoke_on_load=False,
             on_load_failure_callback=raise_extension_exception
--- nova-2015.1.2/tools/config/generate_sample.sh.~1~	2015-10-13 07:52:44.000000000 -0700
+++ nova-2015.1.2/tools/config/generate_sample.sh	2016-01-31 21:59:34.000000000 -0800
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+PATH=/usr/gnu/bin:/usr/bin
+
 print_hint() {
     echo "Try \`${0##*/} --help' for more information." >&2
 }
--- nova-2015.1.2/tools/config/oslo.config.generator.rc.~1~	2015-10-13 07:52:44.000000000 -0700
+++ nova-2015.1.2/tools/config/oslo.config.generator.rc	2016-01-31 22:09:12.591859883 -0800
@@ -1,2 +1,9 @@
-NOVA_CONFIG_GENERATOR_EXTRA_LIBRARIES="oslo.messaging oslo_db oslo.concurrency"
-NOVA_CONFIG_GENERATOR_EXTRA_MODULES=keystonemiddleware.auth_token
+NOVA_CONFIG_GENERATOR_EXTRA_LIBRARIES=" \
+    keystonemiddleware.auth_token \
+    oslo.concurrency \
+    oslo.db \
+    oslo.db.concurrency \
+    oslo.log \
+    oslo.messaging \
+    oslo.middleware \
+"
